charts/redpanda/values.go

```diff
@@ -1010,9 +1010,10 @@ func (t *ExternalTLS) IsEnabled(i *InternalTLS, tls *TLS) bool {
 }
 
 type AdminListeners struct {
-	External ExternalListeners[AdminExternal] `json:"external"`
-	Port     int32                            `json:"port" jsonschema:"required"`
-	TLS      InternalTLS                      `json:"tls" jsonschema:"required"`
+	External    ExternalListeners[AdminExternal] `json:"external"`
+	Port        int32                            `json:"port" jsonschema:"required"`
+	AppProtocol *string                          `json:"appProtocol" jsonschema:"optional"`
```

Comment:
Should this change to
```suggestion
	AppProtocol *string                          `json:"appProtocol,omitempty" jsonschema:"optional"`
```

Reply:
Optional is the default for jsonschema. To remain consistent let's not specify it in the tag.